recipes-debian/iproute2/iproute2.inc

```diff
@@ -0,0 +1,61 @@
+# From: poky/meta/recipes-connectivity/iproute2/iproute2.inc
+# Branch: warrior
+# Rev: 2c7c08232e6757e9b723dc1c8440bae4408c7123
```

Comment:
Shall we use the same format as meta-debian?
https://github.com/meta-debian/meta-debian/blob/warrior/recipes-debian/procps/procps_debian.bb

Reply:
OK. I've created pr #27.